src/compiler/chainrules.jl

```diff
@@ -106,6 +106,7 @@ Convert `x` from the differentials types ChainRules uses to the format Zygote us
 # Zygote convention: even if many AbstractZero partials (i.e. multi-input function), make just 1 nothing.
 @inline wrap_chainrules_output(x::Tuple{Vararg{ChainRules.AbstractZero}}) = nothing
 @inline wrap_chainrules_output(x::ChainRules.AbstractZero) = nothing
+@inlint wrap_chainrules_output(x::ChainRules.NoTangent) = nothing
```

Comment:
This seems completely safe to me.

Re PR's original change  https://github.com/FluxML/Zygote.jl/pull/1075/commits/c53df992fa37faee417db245ca1d7c569d59b201 to propagate nothings, I was surprised this broke no tests. I don't recall now what test motivated that method in the first place, and it's possible that some later change removed the need.